test/csuite/schema_abort/main.c

```diff
@@ -312,6 +311,20 @@ test_cursor(THREAD_DATA *td)
     testutil_check(session->close(session, NULL));
 }
 
+/*
+ * set_flush_tier_delay --
+ *     Set up a random delay for the next flush_tier.
+ */
+static void
+set_flush_tier_delay(WT_RAND_STATE *rnd)
+{
+    /*
+     * We are checkpointing constantly, and we'll do a flush tier with a random delay between 0 -
+     * 10000000 microseconds.
+     */
+    opts->tiered_flush_interval_us = __wt_random(rnd) % 10000001;
```

Comment:
This should be `10 * WT_MILLION + 1` please. For larger numbers we want to use the macros for accuracy.

Reply:
Thanks @sueloverso for pointing it out. I will address this in my next commit.